lib/runner/cli/cli.js

```diff
@@ -207,11 +207,13 @@ class CliRunner {
   }
 
   isTestWorkersEnabled() {
-    return this.test_settings.testWorkersEnabled && !this.singleTestRun();
+    const modules =  Runner.readTestSource(this.test_settings, this.argv);
```

Comment:
`readTestSource` is returning a Promise and it's also already done in the `runTests`. We should use the result from there.

Reply:
Yes, initially I planned to use from `runTests` itself and pass `modules` to `isTestWorkersEnabled`, but in https://github.com/nightwatchjs/nightwatch/blob/7e5edc6b32b00280565d3d2679df73e71b191582/lib/runner/cli/cli.js#L225-L227, `testWorkersMode` is called. Here, also I have to pass `modules` but its parent function `setupConcurrency` is called in `setup` (here, I can't pass modules, as it's called before `runTests`), so I thought of using `readTestSouce` directly again.